tests/migration_test.go

```diff
@@ -321,7 +321,7 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 		Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
 			&expect.BSnd{S: "\n"},
 			&expect.BExp{R: console.PromptExpression},
-			&expect.BSnd{S: "stress --vm 1 --vm-bytes 800M --vm-keep --timeout 1600s&\n"},
+			&expect.BSnd{S: "stress --vm 1 --vm-bytes 8M --vm-keep --timeout 1600s&\n"},
```

Comment:
Does this pass for you locally?
I wonder if 8M is enough to generate enough diry pages to delay live migration...

Reply:
Yes it passes locally.
Anyways, "check-tests-for-flakes" lane has unfortunately failed twice, so I'm not sure this change helps at all